samples/crash/src/main/AndroidManifest.xml

```diff
@@ -16,7 +17,10 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@style/Theme.AppCompat">
+        android:theme="@style/Theme.AppCompat"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        tools:targetApi="s"
```

Comment:
I wonder why we have to use `targetApi` here when the sample browser doesn't need it but still has the same compile version.

Reply:
Attribute dataExtractionRules is only used in API level 31 and higher.